### ISSUE TITLE
Resonance audio fmod plugin works with fmod 2.0?

### ISSUE THREAD

**Issue by sanslash332** [resonance-audio/resonance-audio#23]
Is  the resonance audio fmod plugin fully compatible with the 2.0 releace of the FMOD?

I've tried to use it, but when I get the DSP parameter description of each one of them, instead of get the corresponding name or description the only that i got (after cast it) is a array of unprintable symbols.

This is using fmod v2.0, core api with the c# header, using the resonance audio DLL bundled with that version of FMOD.

If I do the same action, but using fmod 1.10.12 that data display correctly :)

thanks!

**Comment by tristanjl** [comment]
This is an old question, but resonance audio should work with FMOD 2.0 and 2.1 using the version included. It is possible that this issue was fixed in a later version of 2.0.

**Comment by sanslash332** [comment]
Hello @tristanjl 
thanks for your reply!

Effectively, the latest plugins of resonance bundled with fmod works fine, if you know which values you have to pass to each parameter. If you want to check it printing the dspInformation of each parameter, you got the same umprintable symbols.
That is in fmod 2.0; I dont have tried it on 2.1 version.

thanks for your reply!

**Comment by tristanjl** [comment]
Right, looks like there was a fix related to parameter info marshalling for the C# api made for 2.1. It might have also been back ported to 2.08.
